e2e/app/component/create-criteria-modal.ts

```diff
@@ -0,0 +1,37 @@
+import {Page} from 'puppeteer';
+import ClrIconLink from 'app/element/clr-icon-link';
+import {ElementType} from 'app/xpath-options';
+import Dialog from './dialog';
+
+const defaultXpath = '//*[contains(concat(" ", normalize-space(@class), " "), " crit-modal-container ")]';
+
+export enum PhysicalMeasurementsCriteria {
+  BMI = 'BMI',
+  HEIGHT = 'Height',
+  WEIGHT = 'Weight',
+  WAIST_CIRCUMFERENCE = 'Waist Circumference',
+  HIP_CIRCUMFERENCE = 'Hip Circumference',
+  HEART_RATE = 'Heart Rate',
+  BLOOD_PRESSURE = 'Blood Pressure',
+  PREGNANT_ENROLLMENT = 'Pregnant at enrollment',
+  WHEELCHAIR_USER = 'Wheelchair user at enrollment',
+  BP_HYPOTENSIVE = 'Hypotensive (Systolic <= 90 / Diastolic <= 60)', // Blood Pressure
+  BP_NORMAL = 'Normal (Systolic <= 120 / Diastolic <= 80)', // Blood Pressure
+  BP_PREHYPERTENSIVE = 'Pre-Hypertensive (Systolic 121-139 / Diastolic 81-89)', // Blood Pressure
+  BP_HYPERTENSIVE = 'Hypertensive (Systolic >= 140 / Diastolic >= 90)', // Blood Pressure
+  HR_NO_IRREGULARITY = 'No-irregularity detected', // Heart Rhythm Status
+  HR_IRREGULARITY = 'Irregularity detected', // Heart Rhythm Status
+}
+
+
+export default class CreateCriteriaModal extends Dialog {
+
+  constructor(page: Page, xpath: string = defaultXpath) {
+    super(page, xpath);
+  }
+
+  async waitForPhysicalMeasurementCriteriaLink(criteriaType: PhysicalMeasurementsCriteria): Promise<ClrIconLink> {
+    return ClrIconLink.forLabel(this.page, {type: ElementType.Icon, name: criteriaType, iconShape: 'slider'}, this);
```

Comment:
Created this class to demonstrate finding icon-link in modal window. similar to existing function that finds button in dialog.

Reply:
It looks like this class isn't used at all yet? If not, can we not have an example that's also in use? Or maybe wait to merge it?

Reply:
no, not used by any class/function in this PR. To add a usage example, it would cause new files to be added in this PR, outside of scope. It is just for an example on how to find Icon-Link in a modal container.